src/lightning_lite/strategies/ddp.py

```diff
@@ -100,17 +110,18 @@ def process_group_backend(self) -> Optional[str]:
 
     def _configure_launcher(self) -> None:
         assert self.cluster_environment is not None
-        if not self.cluster_environment.creates_processes_externally:
+        if self._start_method == "popen":
             self._launcher = _SubprocessScriptLauncher(self.cluster_environment, self.num_processes, self.num_nodes)
```

Comment:
The `if not self.cluster_environment.creates_processes_externally` is checked already inside `_SubprocessScriptLauncher`